subsys/bluetooth/host/conn.c

```diff
@@ -3331,6 +3353,173 @@ int bt_conn_le_subrate_request(struct bt_conn *conn,
 }
 #endif /* CONFIG_BT_SUBRATING */
 
+#if defined(CONFIG_BT_SHORTER_CONNECTION_INTERVALS)
+int bt_conn_le_read_min_conn_interval_groups(struct bt_conn_le_min_conn_interval_info *info)
+{
+	struct net_buf *rsp;
+	struct bt_hci_op_le_read_min_supported_conn_interval *rp;
+	int err;
+
+	if (info == NULL) {
+		return -EINVAL;
+	}
+
+	err = bt_hci_cmd_send_sync(BT_HCI_OP_LE_READ_MIN_SUPPORTED_CONN_INTERVAL, NULL,
```

Comment:
Future work: This implementation reads this value over HCI. But this value could also be conveyed using device tree when the Controller is known at compile time (`CONFIG_HAS_BT_CTLR`). I think this is worth looking into to save on code size and startup time and possibly RAM as well.